recipes/climpred/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "climpred" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d762b3c715bc77b3f59c8c1ba516ee170b84ca6401099c0cb9ed11dc0d2f4bba
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+
+requirements:
+  host:
+    - python >3.5
+    - pip
+    - setuptools_scm
+    - setuptools_scm_git_archive
+
+  run:
+    - python >3.5
+    - xarray
+    - xskillscore >=0.0.4
+    - numpy
+    - scipy
+    - eofs
+    - netcdf4
+    - matplotlib
```

Comment:
Ahh if this is just a test requirement, you can add it to the `tests: requirements: - matplotlib` section: see https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-requirements

Reply:
I think it was required in the main installation, since it's imported in one of our modules that goes through `__init__.py`. So this does seem like the right spot to put it.

Reply:
Ahh OK!